pgxmock.go

```diff
@@ -106,6 +107,8 @@ type pgxMockIface interface {
 
 	// New Column allows to create a Column
 	NewColumn(name string) *pgproto3.FieldDescription
+
+	ConnInfo() *pgtype.ConnInfo
```

Comment:
Should have a comment to pass linting
```suggestion
	// ConnInfo returns the connection info used for this connection 
	ConnInfo() *pgtype.ConnInfo
```

Reply:
Also please add unit test to cover added code

Reply:
awesome -- thank you @pashagolub